west.yml

```
@@ -72,7 +72,7 @@ manifest:
       revision: 6bb4a17a2f94a39eba295dd55dbeae6f6c4c9b1b
       path: modules/hal/silabs
     - name: hal_st
-      revision: b52fdbf4b62439be9fab9bb4bae9690a42d2fb14
+      revision: 44d5d70b09aeae37c225526e46baa06b5fc79885
```

While it isn't yet merged, you should use the following reference:
```suggestion
      revision: /pull/6/head
```

---

I updated it, but CI still fails.

---

@nikooiko my bad this should be : 
```suggestion
revision: pull/6/head
```